conformance-test-framework/build.gradle

```diff
@@ -18,3 +18,37 @@ dependencies {
     implementation libs.jspecify
     implementation libs.truth
 }
+
+publishing {
+    publications {
+        conformanceTestFramework(MavenPublication) {
+            pom {
+                groupId = 'org.jspecify.conformance'
+                artifactId = 'conformance-test-framework'
```

Comment:
This relates to our repository name discussion. To me, the repetition of 'conformance' in the group and artifact ids doesn't look nice.